q2_demux/tests/test_demux.py

```diff
@@ -939,7 +939,7 @@ def test_single_sample_multiple_files(self):
                 self.temp_dir.name, mode='r')
         with tempfile.TemporaryDirectory() as output_dir:
             # TODO: Remove _PlotQualView wrapper
-            # NEW TODO: Not so sure about this one
+            # NEW TODO: Not sure what happened here
```

Comment:
Just a note, the `TODO` above this, online 941 (`# TODO: Remove _PlotQualView wrapper`) is _super_ old, and does not need to be addressed here.

Reply:
Yeah I figure. When I looked through the diff from your commit the only new todos were removing the old functions and the list. I'm labeling all of mine NEW TODO so I can easily distinguish :joy:. Thank you for the clarification though.

Reply:
We should invent a TODO versioning scheme.

Reply:
Automatically detects when new TODOs are added in a commit and changes the versioning appropriately